components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/workflow/UserSignUpWSWorkflowExecutor.java

```diff
@@ -88,24 +88,22 @@ public WorkflowResponse execute(WorkflowDTO workflowDTO) throws WorkflowExceptio
                 authSchemes.add(HttpTransportProperties.Authenticator.BASIC);
                 auth.setAuthSchemes(authSchemes);
 
-                if(contentType == null){
-                    options.setProperty(Constants.Configuration.MESSAGE_TYPE, HTTPConstants.MEDIA_TYPE_APPLICATION_XML);
+                if (contentType == null) {
+                    options.setProperty(Constants.Configuration.MESSAGE_TYPE, HTTPConstants.MEDIA_TYPE_TEXT_XML);
```

Comment:
As per the offline discussion with malintha this is not an issue.
[How to Determine the SOAP Version of a Message
](http://wso2.com/library/articles/differentiating-between-soap-versions-looking-soap-message/)